parm/aero/obs/config/viirs_n20_aod.yaml

```diff
@@ -0,0 +1,24 @@
+obs space:
+  name: viirs_n20_aod
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX)viirs_n20.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_viirs_n20_$(OBS_DATE).nc4
+  io pool:
+    max pool size: 1
+  simulated variables: [aerosol_optical_depth]
+  channels: 4 
+get values:
+  interpolation method: $(INTERP_METHOD)
+obs operator:
+  name: AodCRTM
+  Absorbers: [H2O,O3]
```

Comment:
Absorbers, clouds and cloud_fraction are not needed/used for AOD, right?

Reply:
I see them in all the yaml files that we've been using, for example:  
      obs operator:
        name: AodCRTM
        Absorbers: [H2O,O3]
        obs options:
          Sensor_ID: v.viirs-m_npp
          EndianType: little_endian
          CoefficientPath: Data/crtm/
          AerosolOption: aerosols_gocart_default

Reply:
ok absorbers yes, but cloud/cloud fraction are not supposed to be included, right?

Reply:
This file doesn't have clouds or cloud fraction: https://github.com/JCSDA-internal/fv3-jedi/blob/develop/test/testinput/3dvar_gfs_aero.yaml it might not matter. Do your test YAMLs include clouds/cloud fraction?

Reply:
right. pushing now.

Reply:
Still need Clouds removed too